src/app/shared/services/dia-backend/asset/dia-backend-asset-repository.service.ts

```diff
@@ -68,15 +82,7 @@ export class DiaBackendAssetRepository {
     );
   }
 
-  async add(proof: Proof) {
-    return this.notificationService.notifyOnGoing(
-      this.createAsset$(proof),
-      this.translocoService.translate('registeringProof'),
-      this.translocoService.translate('message.registeringProof')
-    );
-  }
-
-  private createAsset$(proof: Proof) {
+  createAsset$(proof: Proof) {
```

Comment:
(Only suggestion) I think we should rename this method to `add$(proof: Proof)` for the consistency of the method names of a "Repository".

Reply:
It makes sense since it is still Repository now. I will make changes according to the suggestion.